### DISCUSSION TITLE
Waste Collection stop working

### DISCUSSION THREAD

**Discussion by Pookz42** [mampfes/hacs_waste_collection_schedule#3255] [Q&A] [answered]
Hi there,

integration has suddenly stopped working and shows me unknown. Nothing is displayed in the calendar either. The website and the app work without any problems. The error is displayed as follows for my address: The source has returned an invalid response: “‘NoneType’ object has no attribute ‘capitalize’”. Please check the arguments and try again. When I try other addresses it only does not work with my address. What could be the reason? Has something changed in the website address?

![grafik](https://github.com/user-attachments/assets/6a8cad9b-fdf3-4e3d-8f2b-e71dfa8387b5)

Translated with DeepL.com (free version)

**Comment by 5ila5** [comment]
Unfortunately, I cannot reproduce this issue. Can you try other addresses like 
```json
{"city": "Grefrath",
"district": "Grefrath",
"street": "An Haus Bruch",
"housenumber": 18}
```


If this address work, could you please find another random address that fails

**Comment by Pookz42** [comment]
you can test this:


        city: Schwalmtal
        district: Amern
        street: Hauptstrasse
        housenumber: 9

that didnt work for example

**Comment by Pookz42** [comment]
![grafik](https://github.com/user-attachments/assets/2fdfa705-c031-4370-be90-f6a3363017b2)
![grafik](https://github.com/user-attachments/assets/44ba4eca-8c55-4e8a-b351-e4976d756968)

**Comment by 5ila5** [comment, marked as answer]
Fixed in #3272

They did not provide a bin color for BIO so the source script failed. the source now uses the bin type as fallback.

```json
{"trash_type_colors":{"ppk":"blau","hml":"grau","hml2":null,"lvp":"gelb","bio":null,"wbm":null}}
```


This fix will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>

**Comment by Pookz42** [comment]
Thnaks for your Quick help
